bin/newrelic-admin

```diff
@@ -71,11 +70,15 @@ USAGE
     # we need to make sure there is a command to run
     print_usage('run-perl') unless @ARGV;
 
-    use NewFangle qw(newrelic_init);
+    use NewFangle qw(newrelic_configure_log newrelic_init);
     require NewFangle::App;
     require NewFangle::Agent;
     require NewFangle::Agent::Config;
 
+    my $cfg = NewFangle::Agent::Config->local_settings;
+    newrelic_configure_log( $cfg->{'log_filename'} // 'stdout',
```

Comment:
Why the extra quotes?

Also: why are we defaulting to log to `stdout` instead of `stderr`? It feels like we should be respecting the default value that the config object has. The object should also make sure that this value is never undefined.

It feels like we're doing more work than we need

Reply:
I was running several tests trying to prove that it works, and I think at one point I thought it was not working without the quotes, but it must have been a typo in the name of the key. I've removed them.